betty/project/generate/__init__.py

```diff
@@ -53,25 +53,26 @@ async def generate(project: Project) -> None:
     Generate a new site.
     """
     async with project.app.user.message_progress(
-        _("Generating your site to {output_directory}.").format(
+        _("Generating site to {output_directory}...").format(
```

Comment:
Don't include the directory. It's irrelevant when serving a demo site and in all other cases the path is known beforehand.